src/app/lib/logger.node.js

```diff
@@ -1,107 +1,108 @@
+//     tailable: true,
+//     format: combine(json()),
+//   },
+//   console: {
+//     handleExceptions: true,
+//     humanReadableUnhandledException: true,
+//     level: LOG_LEVEL,
+//     timestamp: true,
+//     format: combine(prettyPrint(), colorize(), consoleLogFormat),
+//   },
+// };
 
 // e.g. outputs 'Article/index.jsx'
-const folderAndFilename = name => {
-  const fileparts = name.split(path.sep);
-  return fileparts.splice(-2).join(path.sep);
-};
-
-const fileLogger = createLogger({
-  format: format.combine(
-    simple(),
-    timestamp({ format: 'YYYY-MM-DD HH:mm:ss.SSS' }),
-
-    // creates a metadata object, that uses our custom formatting
-    metadata({
-      fillExcept: ['timestamp', 'level', 'message'],
-    }),
-  ),
-  transports: [
-    new transports.File(loggerOptions.file),
-
-    // console output is sent to syslog - this can consume a lot of disk space in instances that
-    // handle a lot of traffic, so we only enable console output in some environments
-    ...(process.env.LOG_TO_CONSOLE === 'true'
-      ? [new transports.Console(loggerOptions.console)]
-      : []),
-  ],
-});
+// const folderAndFilename = name => {
+//   const fileparts = name.split(path.sep);
+//   return fileparts.splice(-2).join(path.sep);
+// };
+
+// const fileLogger = createLogger({
+//   format: format.combine(
+//     simple(),
+//     timestamp({ format: 'YYYY-MM-DD HH:mm:ss.SSS' }),
+
+//     // creates a metadata object, that uses our custom formatting
+//     metadata({
+//       fillExcept: ['timestamp', 'level', 'message'],
+//     }),
+//   ),
+//   transports: [
+//     new transports.File(loggerOptions.file),
+
+//     // console output is sent to syslog - this can consume a lot of disk space in instances that
+//     // handle a lot of traffic, so we only enable console output in some environments
+//     ...(process.env.LOG_TO_CONSOLE === 'true'
+//       ? [new transports.Console(loggerOptions.console)]
+//       : []),
+//   ],
+// });
 
 class Logger {
   constructor(callingFile) {
-    createLogDirectory(LOG_DIR);
-    const file = folderAndFilename(callingFile);
+    // createLogDirectory(LOG_DIR);
+    // const file = folderAndFilename(callingFile);
+    const file = callingFile;
 
     this.error = (event, message) => {
-      fileLogger.error({ file, event, message });
+      // console.error({ file, event, message });
     };
 
     this.warn = (event, message) => {
-      fileLogger.warn({ file, event, message });
+      // console.warn({ file, event, message });
     };
 
     this.info = (event, message) => {
-      fileLogger.info({ file, event, message });
+      // console.info({ file, event, message });
     };
 
     this.debug = (event, message) => {
-      fileLogger.debug({ file, event, message });
+      // console.debug({ file, event, message });
     };
 
     this.verbose = (event, message) => {
-      fileLogger.log({ file, event, message });
+      // console.log({ file, event, message });
```

Comment:
Turned all of these into noops because:

1. It was super noisy while doing development. It was hard to debug issues around all the logging.
2. It was also logging to the filesystem via `fs`, which doesn't really work in Vercel land. If we really want to do this right, we'd setup a [log drain](https://vercel.com/blog/log-drains).